node/src/service.rs

```diff
@@ -244,7 +286,10 @@ pub fn new_light(mut config: Configuration) -> Result<TaskManager, ServiceError>
 	let (client, backend, keystore_container, mut task_manager, on_demand) =
 		sc_service::new_light_parts::<Block, RuntimeApi, Executor>(&config)?;
 
-	config.network.notifications_protocols.push(sc_finality_grandpa::GRANDPA_PROTOCOL_NAME.into());
+	config
```

Comment:
Any way to make this not break instantly? Maybe at least after the first one?

Reply:
Looks like it breaks all of them on a new line as soon as it exceeds the max